Kernel/Syscalls/sleep.cpp

```diff
@@ -29,14 +29,20 @@
 
 namespace Kernel {
 
+#define UINTMAX_MAX __UINTMAX_MAX__
```

Comment:
Why not just use `__UINTMAX_MAX__`?

Reply:
I am unsure if SerenityOS will support another compiler, and `__UINTMAX_MAX__` is compiler specific. If and when, a simple `#ifdef` would help.

I also don't like seeing `__` in the main code, seems... ugly.

Reply:
Regarding compiler: That's not a question of compiler, it's a question of which libc the program uses. And since Serenity brings its own LibC, you only need compatibility with [Serenity's LibC](https://github.com/SerenityOS/serenity/blob/master/Libraries/LibC/bits/stdint.h#L75). :D

Regarding ugly: I agree that it's a bit ugly, and you don't need to use it at all. Our `stdint.h` does `#define UINTMAX_MAX __UINTMAX_MAX__`, so you can use `UINTMAX_MAX` directly! :)

Reply:
Crap, using the same libc in userspace and kernel space is epic. I need to get used to it, thanks! (fixed, pushed waiting for CI to give me green light)